### ISSUE TITLE
TypeError: Cannot read property 'pathname' of null

### ISSUE THREAD

**Issue by thomasparsons** [4Catalyzer/found-scroll#13]
I get this error `TypeError: Cannot read property 'pathname' of null` from the most recent version of `found-scroll` (i'm using `"^0.1.2"` - and so it's picking up 0.14 I assume)

I get the error when loading the site/hitting refresh

I'm using the same code as the readme: 
```
const shouldUpdateScrollByPathname = (prevRenderArgs, { location }) => (
  prevRenderArgs && location.pathname !== prevRenderArgs.location.pathname
);
```
and the `prevRenderArgs.location` is returning null, and so failing when trying to hit `pathname`

I've temporarily solved this with an additional check of:

```
prevRenderArgs && prevRenderArgs.location && location.pathname !== prevRenderArgs.location.pathname

```
which isn't super ideal

**Comment by taion** [comment]
i see, will fix

**Comment by taion** [comment]
Fixed in v0.1.5.

BTW, the example was sort of busted. It should be more like https://github.com/4Catalyzer/found-scroll/pull/14/files#diff-04c6e90faac2675aa89e2176d2eec7d8R52, unless you want the initial scroll restore to not work.
